data/mods/CRT_EXPANSION/mutations/crt_wendigo_mutations.json

```diff
@@ -93,7 +93,7 @@
       {
         "attack_text_u": "You ravage %s with your arms",
         "attack_text_npc": "%1$s ravages %2$s with their arm spurs",
-        "body_part": "ARM_R",
+        "body_part": "arm_l",
```

Comment:
```suggestion
        "body_part": "arm_r",
```

Reply:
I added both id's and it works, I guess I forgot to commit it